From 2f1b0477f3dc198d821a852443eb2bea2029c2ba Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Mon, 19 Feb 2024 14:16:42 +0100
Subject: [PATCH 528/841] fix(fdts): change PLL node name in STM32MP25 clock
 tree

Simplify the PLL binding for STM32MP25, change the name of PLL nodes
st,pll@${X} by st,pll-${X+1}.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 21d09a7944915fcbf78536dba763d795597abe47 fix(fdts): change PLL node name in STM32MP25 clock tree]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Idc4127cdd0f1a04afd9fbca49350f2984b37a0c7
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/361015
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
---
 fdts/stm32mp257f-dk-ca35tdcid-rcc.dtsi  | 8 ++++----
 fdts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi | 8 ++++----
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/fdts/stm32mp257f-dk-ca35tdcid-rcc.dtsi b/fdts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
index c3c3b31e9..c3d32e726 100644
--- a/fdts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
+++ b/fdts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
@@ -59,7 +59,7 @@
 		MUX_CFG(MUX_USB2PHY2, MUX_USB2PHY2_FLEX58)
 	>;
 
-	pll1: st,pll@0 {
+	pll1: st,pll-1 {
 		st,pll = <&pll1_cfg_1200Mhz>;
 
 		pll1_cfg_1200Mhz: pll1-cfg-1200Mhz {
@@ -69,7 +69,7 @@
 	};
 
 	/* DRAM clock = 2 * PLL2 clock */
-	pll2: st,pll@1 {
+	pll2: st,pll-2 {
 		st,pll = <&pll2_cfg_600Mhz>;
 
 		pll2_cfg_600Mhz: pll2-cfg-600Mhz {
@@ -79,7 +79,7 @@
 		};
 	};
 
-	pll4: st,pll@3 {
+	pll4: st,pll-4 {
 		st,pll = <&pll4_cfg_1200Mhz>;
 
 		pll4_cfg_1200Mhz: pll4-cfg-1200Mhz {
@@ -88,7 +88,7 @@
 		};
 	};
 
-	pll5: st,pll@4 {
+	pll5: st,pll-5 {
 		st,pll = <&pll5_cfg_532Mhz>;
 
 		pll5_cfg_532Mhz: pll5-cfg-532Mhz {
diff --git a/fdts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi b/fdts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
index 1c0f49098..b98033dc9 100644
--- a/fdts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
+++ b/fdts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
@@ -59,7 +59,7 @@
 		MUX_CFG(MUX_USB2PHY2, MUX_USB2PHY2_FLEX58)
 	>;
 
-	pll1: st,pll@0 {
+	pll1: st,pll-1 {
 		st,pll = <&pll1_cfg_1200Mhz>;
 
 		pll1_cfg_1200Mhz: pll1-cfg-1200Mhz {
@@ -69,7 +69,7 @@
 	};
 
 	/* DRAM clock = 2 * PLL2 clock */
-	pll2: st,pll@1 {
+	pll2: st,pll-2 {
 		st,pll = <&pll2_cfg_600Mhz>;
 
 		pll2_cfg_600Mhz: pll2-cfg-600Mhz {
@@ -78,7 +78,7 @@
 		};
 	};
 
-	pll4: st,pll@3 {
+	pll4: st,pll-4 {
 		st,pll = <&pll4_cfg_1200Mhz>;
 
 		pll4_cfg_1200Mhz: pll4-cfg-1200Mhz {
@@ -87,7 +87,7 @@
 		};
 	};
 
-	pll5: st,pll@4 {
+	pll5: st,pll-5 {
 		st,pll = <&pll5_cfg_532Mhz>;
 
 		pll5_cfg_532Mhz: pll5-cfg-532Mhz {
-- 
2.47.2

